Fixes #451 Removes references to odd number of members in cluster (#470)
hazelcast/hz-docs: docs/modules/fault-tolerance/pages/fault-tolerance.adoc

jobConfig.setSplitBrainProtection(true);
```

> NOTE: Make sure that you disable the split-brain condition when introducing new members to the cluster. Otherwise, both sub-clusters may grow to more than half of the previous size, circumventing split-brain protection.

Reviewer: ```suggestion
NOTE: Make sure that you resolve the split-brain condition before introducing new members to the cluster. Otherwise, both sub-clusters may grow to more than half of the previous size, circumventing split-brain protection.
```

Author: Updated as requested. Also removed invisible characters that were preventing H3s from displaying correctly.
